paper/intro.qmd

```diff
@@ -0,0 +1,7 @@
+
+Several scientific disciplines have widely adopted standards for reporting statistical analyses and results to facilitate their unambiguous interpretation, critical appraisal, and reuse. Indeed, the adoption of such standards has enhanced the transparency of research and has motivated analogous efforts to develop inference (Charniga et al., 2024) and reporting guidelines (nick's paper). Notwithstanding these, there are currently no guidelines that are tailored to the reporting of epidemiological parameters as existing efforts focus on specific parts of the infectious disease pipeline or on best practices for estimating, rather than reporting, them.
+
+In this study, we aim to address this gap. We first summarise common instances of ambiguous reporting of epidemiological parameters, drawn from our experience in infectious disease modelling and reviewing the literature for epidemiological parameters and models across multiple pathogens. We classify poor reporting into two groups - one consisting of information loss, and the second comprising ambiguous information. Information loss occurs when study authors present less than the entirety of the parameter estimates, metadata and contextual information. Ambiguous reporting occurs when there is not enough clarity in the presented estimates, such as not clarifying whether an uncertainty estimate refers to population variability (standard deviation) or the precision of estimator (standard error). In both categories of poor reporting, the reported parameters can be utilised in secondary analyses only if assumptions are made on what has been reported.  We then illustrate through a series of case studies the practical implications of poor reporting and such assumptions on downstream analyses and/or public health policy. Finally, we propose a set of guidelines for clear reporting of epidemiological parameters to prevent ambiguity and ensure consistency and reproducibility.
```

Comment:
I personally prefer to clear differentiate the two aspects of poor reporting with labels. In the original draft I did this with numbers (1), 2)), to me this makes it clearer that these are two core concepts that we will use throughout the paper.

Reply:
What do people think on the inclusion of the algebraic examples of ambiguous reporting? It could just be my personal preference but I find it more easily digestible with examples of what readers would see in other papers than jus text description.

In the original draft:

> such as a $X \pm Y$ where $Y$ could be the standard deviation or standard error, or $X (Y_1 - Y_2)$ where the bounds $Y_1$ and $Y_2$ could be a confidence interval or credible interval

Reply:
>  drawn from our experience in infectious disease modelling and reviewing the literature for epidemiological parameters and models across multiple pathogens

This sentence should cite all of the PERG systematic reviews.